src/components/application_manager/test/hmi_capabilities_test.cc

```diff
@@ -351,57 +574,116 @@ TEST_F(HMICapabilitiesTest, LoadCapabilitiesFromFile) {
   EXPECT_EQ(hmi_apis::Common_AudioType::PCM,
             static_cast<hmi_apis::Common_AudioType::eType>(
                 audio_pass_thru_capabilities_so[0]["audioType"].asInt()));
+}
+
+TEST_F(
+    HMICapabilitiesTest,
+    LoadCapabilitiesFromFile_CheckHmiZoneCapabilities_SuccessLoadAndConvert) {
+  hmi_capabilities_->Init(last_state_wrapper_);
+  const auto hmi_zone_capabilities_so =
+      *(hmi_capabilities_->hmi_zone_capabilities());
 
-  // Check hmi zone capabilities
-  const smart_objects::SmartObject hmi_zone_capabilities_so =
-      *(hmi_capabilities_test->hmi_zone_capabilities());
   EXPECT_EQ(hmi_apis::Common_HmiZoneCapabilities::FRONT,
             static_cast<hmi_apis::Common_HmiZoneCapabilities::eType>(
                 hmi_zone_capabilities_so.asInt()));
+}
 
-  const smart_objects::SmartObject soft_button_capabilities_so =
-      *(hmi_capabilities_test->soft_button_capabilities());
+TEST_F(
+    HMICapabilitiesTest,
+    LoadCapabilitiesFromFile_CheckSoftButtonCapabilities_SuccessLoadAndConvert) {
+  hmi_capabilities_->Init(last_state_wrapper_);
+  const auto& soft_button_capabilities_so =
+      *(hmi_capabilities_->soft_button_capabilities());
+
+  const size_t soft_butons_length = soft_button_capabilities_so.length();
+
+  for (size_t index = 0; index < soft_butons_length; ++index) {
```

Comment:
@iCollin I believe that assert to true value is redundant, because default values could be changed in the preloaded file and you will be needed to update all related Unit tests. It ill be enough to check mandatory parameters for existing in the file.